src/libs/ComponentSpecs.js

```diff
@@ -154,9 +133,9 @@ const prototypeSpecs = {
       }
     }
     self.executeI13nEvent('created', {});
-    if (self.props.scanLinks && self.props.scanLinks.enable) {
-      self._scanLinks();
-    }
+    // if (self.props.scanLinks && self.props.scanLinks.enable) {
+    //   self._scanLinks();
+    // }
```

Comment:
slowly migrate logic to hooks and the new component entry point, comment out logic I already migrated